### PR TITLE
Fixed MacOS keyboard shortcuts

### LINKED ISSUE #1539: Mac keyboard-shortcuts are not written in typical Mac culture
**Documentation/Product Version**
_TinyMCE 4 Documentation_ OR _TinyMCE 5 Documentation_

**Links to Page(s) affected**
https://www.tiny.cloud/docs/advanced/keyboard-shortcuts/

**Description of Issue**
Most Mac keyboards do not have a key label as "Alt", however many of the Mac keyboard-shortcuts refer to the "Alt" key.

**Suggested Fix**
Probably what is meant is the "Option" key. Consider replacing references to "Alt" with "Option".

### PR DESCRIPTION
**Pull request by tylerkelly13**
Related Ticket: #1539 

Description of Changes:
* Updated the MacOS keyboard shortcuts reference
* Added missing shortcuts
* Formatted table markup
* Fixed incorrect shortcuts
* ~Moved accessibility shortcuts in the first table to the accessibility shortcut table~
* Updated shortcut descriptions to improve clarity
* fixes #1539 

Pre-checks:
- [x] Branch prefixed with `feature/` or `hotfix/`
- [x] `_data/nav.yml` has been updated (if applicable)
- [x] Files has been included where required (if applicable)
- [x] Files removed have been deleted, not just excluded from the build (if applicable)

Review:
- [x] Documentation Team Lead has reviewed
- [x] Product Manager has reviewed

### CODE REVIEW

File: advanced/keyboard-shortcuts.md [hunk 1]
```diff
@@ -11,47 +11,51 @@ keywords: keyboard shortcuts
...
+| Focus to toolbar                             | Alt+F10               | Option+F10                  | core                                                       |
+| Focus to element path                        | Alt+F11               | Option+F11                  | core                                                       |
+| Focus to contextual toolbar                  | Ctrl+F9               | Ctrl+F9                     | core                                                       |
+| Open the popup menu on split toolbar buttons | Shift+Enter           | Shift+Enter                 | core                                                       |
+| Open the help dialog                         | Alt+0                 | Option+0                    | [help]({{ site.baseurl }}/plugins/help/)                   |
+| Insert link                                  | Ctrl+K                | Command+K                   | [link]({{ site.baseurl }}/plugins/link/)                   |
+| Toggle Fullscreen                            | Ctrl+Shift+F          | Ctrl+Shift+F                | [fullscreen]({{ site.baseurl }}/plugins/fullscreen/)       |
```
Comment (lnewson) [inline advanced/keyboard-shortcuts.md:38]: This is incorrect, it should be `Command+Shift+F` for Mac.

File: advanced/keyboard-shortcuts.md [hunk 2]
```diff
@@ -11,47 +11,51 @@ keywords: keyboard shortcuts
...
-| Close submenu | Left Arrow |
-| Close dialog | Esc |
-| Close menu | Esc |
-| Move focus back to editor body | Esc |
+| Action                         | Key                               |
+|--------------------------------|-----------------------------------|
+| Execute command                | Enter / Space                     |
+| Focus on next UI Component     | Tab                               |
+| Focus on previous UI Component | Shift+Tab                         |
+| Focus next widget              | Right Arrow / Tab / Down Arrow    |
```
Comment (lnewson) [inline advanced/keyboard-shortcuts.md:51]: I'm not sure widget is a good name here, maybe this should be `UI Component` as well?

File: advanced/keyboard-shortcuts.md (hunk 1 above)
Comment (tylerkelly13) [reply advanced/keyboard-shortcuts.md]: Fixed by 467979ea69821ffb840fd6066847fe75d513b82b

File: advanced/keyboard-shortcuts.md (hunk 2 above)
Comment (tylerkelly13) [reply advanced/keyboard-shortcuts.md]: "widget" is a terrible name there. So I replaced it, also tested these shortcuts, they were wrong. Updated.

File: advanced/keyboard-shortcuts.md [hunk 3]
```diff
@@ -11,47 +11,51 @@ keywords: keyboard shortcuts
...
+| Focus to toolbar                                                                 | Alt+F10                  | Option+F10               |
+| Focus to element path                                                            | Alt+F11                  | Option+F11               |
+| Focus to contextual toolbar                                                      | Ctrl+F9                  | Ctrl+F9                  |
+| Focus on next UI Element<br>(such as: Menu bar, Toolbar, Toolbar Group, Status Bar Item)     | Tab                      | Tab                      |
+| Focus on previous UI Element<br>(such as: Menu bar, Toolbar, Toolbar Group, Status Bar Item) | Shift+Tab                | Shift+Tab                |
+| Focus next Control<br>(such as: toolbar button, menu, or menu item)                  | Right Arrow / Down Arrow | Right Arrow / Down Arrow |
+| Focus previous Control<br>(such as: toolbar button, menu, or menu item)              | Left Arrow / Up Arrow    | Left Arrow / Up Arrow    |
+| Open menu, toolbar menu button, split toolbar button, or group toolbar button    | Down Arrow               | Down Arrow               |
```
Comment (EkimChau) [inline advanced/keyboard-shortcuts.md:52]: Down Arrow || Spacebar

File: advanced/keyboard-shortcuts.md [hunk 4]
```diff
@@ -11,47 +11,51 @@ keywords: keyboard shortcuts
...
+| Open the popup menu on split toolbar buttons                                     | Shift+Enter              | Shift+Enter              |
+| Open submenu                                                                     | Right Arrow              | Right Arrow              |
+| Close submenu                                                                    | Left Arrow               | Left Arrow               |
```
Comment (EkimChau) [inline advanced/keyboard-shortcuts.md:55]: Left Arrow || Escape

File: advanced/keyboard-shortcuts.md (hunk 3 above)
Comment (tylerkelly13) [reply advanced/keyboard-shortcuts.md]: ok, I had to split this item out, split toolbar and group toolbar buttons have unique behaviour. See: https://github.com/tinymce/tinymce-docs/pull/1543/commits/b294e937ae83eb64451f079631b6b16d17bfaac8

File: advanced/keyboard-shortcuts.md (hunk 4 above)
Comment (tylerkelly13) [reply advanced/keyboard-shortcuts.md]: Fixed in https://github.com/tinymce/tinymce-docs/pull/1543/commits/b294e937ae83eb64451f079631b6b16d17bfaac8

**Pull request merged**